src/scenes/home/landing/emailSignup/emailSignup.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import Section from 'shared/components/section/section';
+import axios from 'axios';
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import config from 'config/environment';
+import Form from 'shared/components/form/form';
+import FormEmail from 'shared/components/form/formEmail/formEmail';
+import FormButton from 'shared/components/form/formButton/formButton';
+import styles from './emailSignup.css';
+
+class EmailSignup extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      emailValid: false,
+      success: false,
+      isLoading: false
+    };
+  }
+
+  onEmailChange = (value, valid) => {
+    this.setState({ email: value.toLowerCase(), emailValid: valid });
+  }
+
+  handleOnClick = (e) => {
+    e.preventDefault();
+
+    this.setState({ isLoading: true });
+
+    if (this.isFormValid()) {
+      const { email } = this.state;
+      axios.post(`${config.backendUrl}/email_list_recipients`, {
+        user: { email }
```

Comment:
@Cooperbuilt - I do not get any backend errors (i.e. `Could not find public_suffix-3.0.0 in any of the sources`).

I do, though, get this error:

![image](https://user-images.githubusercontent.com/7482329/34179577-c74dfc16-e4c8-11e7-96b2-44e7aa587ff6.png)


The param the [backend is expecting](https://operationcodeapi.docs.apiary.io/#reference/0/email-list-recipients-create/add-a-guest-(non-user)-to-the-oc's-email-list) is called `email`.   i.e.:

``` 
/api/v1/email_list_recipients?email=jack@example.com
```

Are these API endpoint lines resolving to that?  Or is `user` being folded in, too?